pkg/errs/errno.go

```diff
@@ -52,6 +52,7 @@ var (
 	ErrLoadKeyspaceGroupsTimeout       = errors.Normalize("load keyspace groups timeout", errors.RFCCodeText("ErrLoadKeyspaceGroupsTimeout"))
 	ErrLoadKeyspaceGroupsTerminated    = errors.Normalize("load keyspace groups terminated", errors.RFCCodeText("ErrLoadKeyspaceGroupsTerminated"))
 	ErrLoadKeyspaceGroupsRetryExhaustd = errors.Normalize("load keyspace groups retry exhausted, %s", errors.RFCCodeText("ErrLoadKeyspaceGroupsRetryExhaustd"))
+	ErrKeyspaceNotAssigned             = errors.Normalize("the keyspace isn't assigned to any keyspace group, %s", errors.RFCCodeText("PD:tso:ErrKeyspaceNotAssigned"))
```

Comment:
Where do we use it?

Reply:
Changed to

Reply:
Made change (check **return pdpb.Timestamp{}, keyspaceGroupID, errs.ErrKeyspaceNotAssigned.FastGenByArgs(keyspaceID)**):
```
	am, err := kgm.GetAMWithMembershipCheck(keyspaceID, keyspaceGroupID)
	if err != nil {
		// The keyspace doesn't belong to this keyspace group, we should check if it belongs to any other
		// keyspace groups, and return the correct keyspace group ID to the client.
		for i := 0; i < int(mcsutils.MaxKeyspaceGroupCountInUse); i++ {
                         // return if the keyspace is found in the keyspace lookup tables.
		}
		return pdpb.Timestamp{}, keyspaceGroupID, errs.ErrKeyspaceNotAssigned.FastGenByArgs(keyspaceID)
	}
```